packages/cli/src/commands/db.ts

```diff
@@ -44,11 +44,13 @@ async function runSeed() {
   log.success("Done seeding")
 }
 
+export interface Flags {
+  file: boolean
+}
+
 export class Db extends Command {
   static description = `Run database commands
-${require("chalk").bold(
-  "seed",
-)}   Generates seeded data in database via Prisma 2. You need db/seeds.ts or db/seeds/index.ts.
+${require("chalk").bold("seed")}   Generates seeded data in database via Prisma 2.
```

Comment:
Since Prisma now follows semver
```suggestion
${require("chalk").bold("seed")}   Generates seeded data in database via Prisma.
```